.github/workflows/ci.yml

```diff
@@ -0,0 +1,147 @@
+      - name: 'CMake build'
+        run: |
+          cmake --build build --parallel
+      - name: 'CTest'
+        run: |
+          cd build
+          ctest -V -C ${{ matrix.config }}
+      - name: 'Install'
+        run: |
+          cmake --build build --target install
+      - name: 'Test installed flatcc prefix'
+        run: |
+          mkdir user && cd user
+          cmake "${{ github.workspace }}/test/install_test" -DCMAKE_PREFIX_PATH="${{ github.workspace }}/prefix"
+          cmake --build .
+          ctest .
+  test_windows:
+    name: 'Test Windows with CMake'
+    if: ${{ github.ref != 'refs/heads/ci-more' }}
+    strategy:
+      matrix:
+        config: ['Release', 'Debug']
+        arch: ['Win32', 'x64']
+    runs-on: 'windows-latest'
+    steps:
+      - name: 'Checkout repo'
+        uses: actions/checkout@v2
+      - name: 'CMake configure'
+        run: |
+          mkdir build
+          cd build
+          cmake .. -A ${{ matrix.arch }} -DFLATCC_INSTALL=ON -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/prefix
+      - name: 'CMake build'
+        run: |
+          cmake --build build --parallel --config ${{ matrix.config }} --verbose
+      - name: 'CTest'
+        run: |
+          cd build
+          ctest -V -C ${{ matrix.config }}
+      - name: 'Install'
+        run: |
+          cmake --build build --target install --config ${{ matrix.config }}
+      - name: 'Test installed flatcc prefix'
+        run: |
+          mkdir user && cd user
+          cmake "${{ github.workspace }}/test/install_test" -A ${{ matrix.arch }} `
+              -DCMAKE_PREFIX_PATH="${{ github.workspace }}/prefix"
+          cmake --build . --config ${{ matrix.config }}
+          ctest . -C ${{ matrix.config }}
+  test_mingw_on_linux:
+    name: 'Test cross building to Windows with CMake'
+    if: ${{ github.ref != 'refs/heads/ci-more' }}
+    strategy:
+      matrix:
+        config: ['Release']
+        arch: ['x64']
+        shared_build: ['ON']
+        config_build: ['Release']
+        shared_host: ['ON']
+        config_host: ['Release']
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: 'Install mingw'
+        run: |
+          sudo apt-get install mingw-w64
+      - name: 'Checkout repo'
+        uses: actions/checkout@v2
+      - name: 'CMake configure/build/install build flatcc (native)'
+        run: |
+          mkdir build_native
+          cd build_native
+          cmake .. -DBUILD_SHARED_LIBS=${{ matrix.shared_build }} -DCMAKE_BUILD_TYPE=${{ matrix.config_build }} \
+              -DFLATCC_INSTALL=ON -DCMAKE_INSTALL_PREFIX=${{ github.workspace }}/prefix_build
+          cmake --build . --parallel
+          cmake --build . --target install
+      - name: 'CMake configure/build/install host flatcc (cross)'
+        run: |
+          mkdir build_mingw
+          cd build_mingw
+          cmake .. -DCMAKE_TOOLCHAIN_FILE=../cmake/x86_64-w64-mingw32-toolchain.cmake -DFLATCC_ALLOW_WERROR=OFF \
```

Comment:
I had to disable `-Werror` for mingw because there were `-Wformat` warnings when building for mingw.
See https://github.com/madebr/flatcc/runs/1591429817?check_suite_focus=true for the output.

Reply:
That's no good. And there is actually a problem in the source code.

` return (size_t)sprintf(literal, "%s(%llu)", cast, llu(value->u));`

The should be %PRIu64 or something, not llu. That is old.

More generally, if there is a really good reason, the main CMakeList.txt file disables specific warning types for specific compiler versions, or the warnings are disabled in a file always included in the build, but that is mostly when compilers consistently breaks standards compliant code, not mentioning any names, but there are some MSVC warnings there: https://github.com/dvidelabs/flatcc/blob/master/include/flatcc/portable/pwarnings.h

Reply:
fixed on master, please try with warnings.

Reply:
See https://github.com/madebr/flatcc/runs/1591699375?check_suite_focus=true

Reply:
This is more or less unfixable from a quick internet search. Mingw32 does not like I64u I64d modifiers which it itself defines PRI macros to in inttypes.h, but it also doesn't like %llu and %lld. Compiling runtime only might solve it.
I'd patch inttypes.h if I could, the machinery is there, but the solution is not, it seems.

Reply:
If not already applied, could you try one or several of
-DFLATCC_PORTABLE
-std=c99
-std=c11

Reply:
I've applied the suggestion from https://stackoverflow.com/questions/44382862/how-to-printf-a-size-t-without-warning-in-mingw-w64-gcc-7-1:
`-Wno-pedantic-ms-format`

When building local, the error moves to `reflection_test`, which contains another `%lld`.

Reply:
My awesome ag search powers have failed me.

Reply:
Fixed in master

Reply:
YES!
It's all-green once again!
https://github.com/madebr/flatcc/actions/runs/436827350

Reply:
👍